### PR TITLE
Added the AI-on-the-edge-device

### PR DESCRIPTION
**Pull request by caco3**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

The Discovery functionality will come in the next weeks with the next release (>12.0.1) but is already available in the [rolling](https://github.com/jomjol/AI-on-the-edge-device/tree/rolling) branch.

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_docs/energy/electricity-grid.markdown [hunk 1]
```diff
@@ -45,6 +45,11 @@ We have developed [Home Assistant Glow](https://github.com/klaasnicolaas/home-as
 
 ![Photo of Home Assistant Glow attached to an electricity meter](/images/docs/energy/home-assistant-glow.jpg)
 
+#### Read the meter using an AI-on-the-edge-device
+[AI-on-the-edge-device](https://github.com/jomjol/AI-on-the-edge-device) is a project running on an ESP32-CAM and can be fully integrated into Home Assistant using the Home Assistant Discovery Functionality resp. MQTT. It digitalizes your gas/water/electricity meter display and provides its data through various ways.
```
Comment (frenck) [inline source/_docs/energy/electricity-grid.markdown]:
```suggestion
#### Read the meter using an AI-on-the-edge-device

[AI-on-the-edge-device](https://github.com/jomjol/AI-on-the-edge-device) is a project running on an ESP32-CAM and can be fully integrated into Home Assistant using the Home Assistant discovery functionality of MQTT. It digitalizes your gas/water/electricity meter display and provides its data in various ways. 
```

File: source/_docs/energy/electricity-grid.markdown [hunk 2]
```diff
@@ -45,6 +45,11 @@ We have developed [Home Assistant Glow](https://github.com/klaasnicolaas/home-as
 
 ![Photo of Home Assistant Glow attached to an electricity meter](/images/docs/energy/home-assistant-glow.jpg)
 
+#### Read the meter using an AI-on-the-edge-device
+[AI-on-the-edge-device](https://github.com/jomjol/AI-on-the-edge-device) is a project running on an ESP32-CAM and can be fully integrated into Home Assistant using the Home Assistant Discovery Functionality resp. MQTT. It digitalizes your gas/water/electricity meter display and provides its data through various ways. 
+
+![image](https://user-images.githubusercontent.com/1783586/201498709-fafd7f64-d131-42c9-bf89-3d990cbadf33.png)
```
Comment (frenck) [inline source/_docs/energy/electricity-grid.markdown]: - Images need to be included in this repository, we don't allow external images.
- Please provide a meaningful alt description.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_docs/energy/electricity-grid.markdown (hunk 2 above)
Comment (caco3) [reply source/_docs/energy/electricity-grid.markdown]: Thanks for looking into it.

I adjusted it according your review.

I could not figure out how to test if the image URL is correct, so I just made it like the other images on that page.

**Comment by caco3** [comment]
@frenck, I updated according your review. Also the Release 13.0.1 is now out, containing MQTT Discovery.

So could you have another review?

**Pull request closed without merging**